site2/docs/administration-isolation-bookie.md

```diff
@@ -10,24 +10,230 @@ import TabItem from '@theme/TabItem';
 ````
 
 
-A namespace can be isolated into user-defined groups of bookies, which guarantees all the data that belongs to the namespace is stored in desired bookies. The bookie affinity group uses the BookKeeper [rack-aware placement policy](https://bookkeeper.apache.org/docs/latest/api/javadoc/org/apache/bookkeeper/client/EnsemblePlacementPolicy.html) and it is a way to feed rack information which is stored as JSON format in znode.
+Isolating bookies equals isolating message storage, which is a data storage mechanism that provides isolation and safety for specific topics. 
 
-You can set a bookie affinity group using one of the following methods.
+Bookie isolation is controlled by BookKeeper clients. For Pulsar, there are two kinds of BookKeeper clients to read and write data. 
+*  BookKeeper clients on the broker side
+  Pulsar brokers use these BookKeeper clients to read and write topic messages. 
```

Comment:
I guess this is not the expected layout. Do you want to add a period or ... ?

<img width="529" alt="image" src="https://user-images.githubusercontent.com/50226895/185006297-6127b418-cb3e-4408-825d-57439d11052e.png">